aws/ci_build.rake

```diff
@@ -138,16 +138,6 @@ task :apps_task do
   packager.decompress_package(package)
 end
 
-# TODO: This is a temporary task meant to cleanup old code-studio packages.
-# It should go away in the nearish future
-task :code_studio_task do
-  HipChat.log 'Cleaning code-studio package'
-  # get rid of any symlink to a built package
-  RakeUtils.system 'rm', dashboard_dir('public/code-studio') if File.exist?(dashboard_dir('public/code-studio'))
-  # also delete the package itself
-  RakeUtils.system 'rm -rf', dashboard_dir('public/code-studio-package') if File.exist?(dashboard_dir('public/code-studio-package'))
-end
-
```

Comment:
I've been meaning to do this for a bit but haven't had a chance. The main thing I'd be worried about for this particular removal is that the right thing happens when creating an adhoc. I think in that case we start with an image that still has code-studio paths, and needs a cleanup step. Might be that part of the answer is to update our starting image.